webapp/channels/src/components/external_image/external_image.tsx

```diff
@@ -1,43 +1,30 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React from 'react';
+import type {ReactElement} from 'react';
+import {memo} from 'react';
 
 import type {PostImage} from '@mattermost/types/posts';
 
 import {getImageSrc} from 'utils/post_utils';
 
+import {isSVGImage} from './external_image_isSVGImage';
+
 interface Props {
-    children: (src: string) => React.ReactNode;
+    children: (src: string) => ReactElement | null;
```

Comment:
Why this change? `React.ReactNode` should be able to handle null.

Reply:
Initially, I was getting type errors because I was using memo. When I replaced it with ReactElement, it fixed the ts error.

However when I ran ```make test``` Snapshot test in other component had External Image was failing. 

I have since reverted it back. to React.ReactNode

Reply:
Oh,  I see. Then it should be as simple as to wrap the returned children in a react fragment. Something like...
```
return (<>
    {props.children(src);}
</>);
```

Reply:
nice. changes made